app/assets/javascripts/discourse/app/components/sidebar/section-form-link.js

```diff
@@ -44,25 +45,27 @@ export default class SectionFormLink extends Component {
       this.dragCount === 0 &&
       (this.dragCssClass === "drag-above" || this.dragCssClass === "drag-below")
     ) {
-      this.dragCssClass = null;
+      discourseDebounce(
+        this,
+        () => {
```

Comment:
Is it really debounced here? Seems unlikely given it's'an anonymous function

Reply:
Yeah, it feels hacky. What I wanted to achieve is to delay clear CSS. When we move mouse between draggable elements, we switch from bottom line for top draggable element to top border for bottom draggable element.

Without delay

https://github.com/discourse/discourse/assets/72780/bd5c2283-7792-48d7-9b48-e1c7078115eb

With delay

https://github.com/discourse/discourse/assets/72780/541dcf15-3ce8-4f99-9162-3b5ab10c537d

Reply:
This is not what I meant. You can't debounce an anonymous function, js can't know it's the same function given you create a new anonymous one each time.